aqua/barrier/Portfile

```diff
@@ -46,15 +44,33 @@ if { ${os.major} < 16 } {
     configure.args-append \
```

Comment:
this needs the `os.platform` check as well

Reply:
please make this last requested change and we can merge

Reply:
Done